gallery/visualtests/nxTag.js

```diff
@@ -57,6 +60,21 @@ describe('NxTag', function() {
 
       await checkScreenshot(targetElement);
     });
+
+    it('has a plus icon when not selected', async function() {
+      const [targetElement] = await waitAndGetElements(selector);
+
+      await checkScreenshot(targetElement);
+    });
+
+    it('has a times icon when selected', async function() {
```

Comment:
Aren't these icons covered in the screenshots of the "by default" and "when clicked" tests above?

Reply:
> Aren't these icons covered in the screenshots of the "by default" and "when clicked" tests above?

Oh yes, removed the redundant test here: c153bbd5e4583c645d8ad17368aa71494ab126b4